### PR TITLE
AzureDataFactory: Add RestSink in Models

### PR DESCRIPTION
**Pull request by kalyanrk61**

### CODE REVIEW

**Comment by isra-fel** [comment]
@kalyanrk61  Could you
- Add a link to the openapi spec review PR here
- Check in the metadata TXT file generated when you run `generate.ps1`[(details)](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-generation-from-updated-spec)
- Update api tag in `AzSdk.RP.props`, release notes in csproj
- Update versions in csproj and `assemblyinfo.cs` if you are going to release

Thanks

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Generated/Models/RestSink.cs [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Management.DataFactory.Models
```
Comment (kalyanrk61) [inline sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Generated/Models/RestSink.cs:11]: Forgot to add the tests, will add it and sent an update.

**Comment by kalyanrk61** [comment]
> @kalyanrk61 Could you
> 
> * Add a link to the openapi spec review PR here
> * Check in the metadata TXT file generated when you run `generate.ps1`[(details)](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-generation-from-updated-spec)
> * Update api tag in `AzSdk.RP.props`, release notes in csproj
> * Update versions in csproj and `assemblyinfo.cs` if you are going to release
> 
> Thanks

Spec PR:
https://github.com/Azure/azure-rest-api-specs/pull/10186

will look into the other comments

**Comment by kalyanrk61** [comment]
> > @kalyanrk61 Could you
> > 
> > * Add a link to the openapi spec review PR here
> > * Check in the metadata TXT file generated when you run `generate.ps1`[(details)](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-generation-from-updated-spec)
> > * Update api tag in `AzSdk.RP.props`, release notes in csproj
> > * Update versions in csproj and `assemblyinfo.cs` if you are going to release
> > 
> > Thanks
> 
> Spec PR:
> [Azure/azure-rest-api-specs#10186](https://github.com/Azure/azure-rest-api-specs/pull/10186)
> 
> will look into the other comments

trying to get the required PS modules installed with following command and got failures, any suggestion on how to fix that.
msbuild eng/mgmt.proj /t:Util /p:UtilityName=InstallPsModules

Failure: 
"C:\Source\azure-sdk-for-net\eng\mgmt.proj" (Util target) (1) ->
  C:\Source\azure-sdk-for-net\eng\mgmt\Directory.Build.Mgmt.props(10,78): error MSB4066: The attribute "Sdk" in element
 <Import> is unrecognized. [C:\Source\azure-sdk-for-net\eng\mgmt.proj]

Thanks
Kalyan.

**Comment by isra-fel** [comment]
Never saw that error before 🤔  A few things to try:

1. Make sure you are using latest VS2019. What's your version of msbuild?
1. Clean `C:\Users\***\.nuget` and try again

**Comment by kalyanrk61** [comment]
> Never saw that error before 🤔 A few things to try:
> 
> 1. Make sure you are using latest VS2019. What's your version of msbuild?
> 2. Clean `C:\Users\***\.nuget` and try again

Thank you! @isra-fel , this PR will be taken care by ADMS team in china they have new process for SDK chagnes, I am closing this PR.

**Pull request closed without merging**